RecipesLyricsMashup/RecipesLyricsMashupV2/RecipeLyricsMashup/RecipeLyricsMashup/Pages/Index.cshtml

```diff
@@ -14,47 +15,51 @@
 </div>
 
 
-<form action="" method="GET">
+<form action="" method="GET" >
 
-    First name: <input type="text" name="search">
+    First name: <input type="text" name="search" value=@search> @*Morgan added "value" tag to repopulate the field with the last searched string value*@
 
     <input type="submit" value="Go" />
 
 </form>
 
-<table>
-    <tr>
-        <th>Title</th>
-        <th>Ingredients</th>
-        <th>Link</th>
-    </tr>
-    @{ 
-        foreach (QuickTypeRecipe.Result result in recipePuppyJson)
+<table class="table table-striped"> @*Morgan added*@
+    <thead class="thead-dark"> @*Morgan added*@

```

Comment:
Ditto comment above... no need to attribute code to yourself; we can see this in GitHub history.  Adding that in code is clutter.

Reply:
Bootstrap themes are nice.